src/commands/synthetics/utils/public.ts

```diff
@@ -281,6 +281,8 @@ const getBatch = async (api: APIHelper, trigger: Trigger): Promise<Batch> => {
 
 const getTestByPublicId = (id: string, tests: Test[]): Test => tests.find((t) => t.public_id === id)!
 
+export const normalizePublicId = (id: string) => id.match(PUBLIC_ID_REGEX)?.[0]
```

Comment:
I thought we had an ESLint rule about that, but for `export`ed functions, we always want to have an explicit return type

Reply:
ah wait never mind i thing i was doing it wrong 😂